### DISCUSSION TITLE
Low contrast between dark colors

### DISCUSSION THREAD

**Discussion by gigawatts** [jesec/flood#213] [Ideas]
I find the light grey/blue on dark grey/blue color scheme of the left side panel of the main UI to be very difficult to read, there is not enough contract between these two colors, and causes eye strain to read.

It would be awesome to see an extra setting in the Settings > User Interface to pick between a couple different color schemes, or pick a background color, or something along those lines.

It's also just a little jarring to jump into the Settings menu view, and get a "dark mode" view, where the majority of the main view is "light mode".

As a temporary fix to this issue, I am using the following Tampermonkey script (in Chrome) to change a couple class styles to lighten up the background colors of some of the left side panel elements, making them much easier to read:

```
// ==UserScript==
// @name         Transmission Web UI Flood color change
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  Fix the low contract colors used on the Flood web UI
// @author       You
// @match        http://*/transmission/web/*
// @grant        GM_addStyle
// @run-at document-start
// ==/UserScript==

GM_addStyle (`
/* Main view left panel background color */
div.svelte-j1ddzx {
  background: #f1f7ff !important;
}

/* Main view left panel badge background color */
.badge.svelte-1polwkd {
  background-color: #cee2ff !important;
}

/* Settings and Detail view background color */
.content.svelte-18jciuz {
  background: #eaf3ff !important;
}

/* Detail view H1 header text color */
h1.svelte-v24s2w {
  color: #3ea7ff !important;
}

/* Settings main heading text color */
.menu.svelte-q2nims {
  color: #000 !important;
}

/* Settings h2 text color */
h2.svelte-1kue5c1 {
  color: #000 !important;
}

/* Settings menu input box background color */
input.svelte-ihlvjc.svelte-ihlvjc {
  background: #eaf3ff !important;
}
`);
```

**Comment by jesec** [comment]
Um... I don't think there is such style in Flood. Plus, Flood does not use svelte. I think you are probably using @johman10's [clone](https://github.com/johman10/flood-for-transmission).

**Reply by johman10** [reply]
Thanks for the ping! :)

**Comment by johman10** [comment]
That definitely looks like my project instead, you can find it here: https://github.com/johman10/flood-for-transmission.

Note that someone has already reported this issue on my repo as well: https://github.com/johman10/flood-for-transmission/issues/54. Also note that Dark mode is coming to flood-for transmission as well. I'm working on some color fixes however it's quite slow process. Both issues on the flood-for-transmission repo would probably resolve your issues though. :)

Thanks for reporting though but let's continue the discussion in the right repository. :)

**Comment by gigawatts** [comment]
Whoops, sorry I hit the wrong project, I was linked here from [this project](https://haugene.github.io/docker-transmission-openvpn/config-options/). I'll have to hit up the owner of that project to correct their link to the right repo for flood.

Thank you for pointing me in the right direction @jesec you may close this issue.
